custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -140,15 +141,19 @@ def updates(self):
             cases_ccs_lactating = ut.cases_ccs_lactating,
             cases_ccs_pregnant_all = ut.cases_ccs_pregnant_all,
             cases_ccs_lactating_all = ut.cases_ccs_lactating_all,
-            cases_person_beneficiary_v2 = COALESCE(cases_person_beneficiary_v2, 0) + ut.cases_ccs_pregnant + ut.cases_ccs_lactating
+            cases_person_beneficiary_v2 = COALESCE(cases_person_beneficiary_v2, 0) + ut.cases_ccs_pregnant + ut.cases_ccs_lactating,
+            valid_home_visits = ut.valid_home_visits,
+            expected_home_visits = ut.expected_home_visits
         FROM (
             SELECT
                 awc_id,
                 month,
                 sum(pregnant) AS cases_ccs_pregnant,
                 sum(lactating) AS cases_ccs_lactating,
                 sum(pregnant_all) AS cases_ccs_pregnant_all,
-                sum(lactating_all) AS cases_ccs_lactating_all
+                sum(lactating_all) AS cases_ccs_lactating_all,
+                sum(valid_visits) AS valid_home_visits,
```

Comment:
I think it could be a little confusing that the name changes here. Better to stay consistent for easier code grepping